grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy

```diff
@@ -142,19 +142,21 @@ class OccurrenceTagLib {
             span(class: "active-filters__label") {
                 if (item.key.contains("occurrence_year")) {
                     fqLabel = fqLabel
-                        .replaceAll(':',': ')
+                        .replaceAll(':', ': ')
                         .replaceAll(
                             'occurrence_year',
                             alatag.message(code: 'facet.occurrence_year', default:'occurrence_year')
                         )
+                        .replaceAll(/(\d{4})\-.*?Z/)
 
-                    mkp.yieldUnescaped(fqLabel.replaceAll(/(\d{4})\-.*?Z/) { all, year ->
+                    mkp.yieldUnescaped(fqLabel { all, year ->
                         def year10 = year?.toInteger() + 10
```

Comment:
isn't this going into exception if year can't be integerized?

Reply:
Nick has probably handled that somewhere

Reply:
ALA policy on input validation is at best "crash with 500"

Reply:
It's this way for logging purposes